File: GymFit_FE/src/types/subscriptionPlan.ts
export interface SubscriptionPlan {
    Id: number;
    Name: string;
    Description: string;
    Price: number;
    DurationInDays: number;
    Type: string;
}

export enum SubscriptionType {
    Fitness = 0,
    Pilates = 1,
    Yoga = 2,
    CrossFit = 3,
    Swimming = 4,
    PersonalTraining = 5,
    Cardio = 7
} 

File: GymFit_BE/Models/SubscriptionPlan.cs

public class SubscriptionPlan
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public decimal Price { get; set; }
    public int DurationInDays { get; set; }
    public SubscriptionType Type { get; set; }
    public bool IsActive { get; set; }
}